components/core/tests/test-ir_encoding_methods.cpp

```diff
@@ -582,25 +638,16 @@ TEMPLATE_TEST_CASE(
     ));
     size_t const encoded_preamble_end_pos = ir_buf.size();
 
-    string message;
-    epoch_time_ms_t ts;
     vector<string> reference_messages;
     vector<epoch_time_ms_t> reference_timestamps;
-
-    // First message:
-    message = "Static <\text>, dictVar1, 123, 456.7, dictVar2, 987, 654.3, end of static text";
-    ts = get_next_timestamp_for_test<TestType>();
-    REQUIRE(encode_message<TestType>(ts, message, logtype, ir_buf));
-    reference_messages.push_back(message);
-    reference_timestamps.push_back(ts);
-
-    // Second message:
-    message = "Static <\text>, dictVar3, 355.2352512, 23953324532112, "
-              "python3.4.6, end of static text";
-    ts = get_next_timestamp_for_test<TestType>();
-    REQUIRE(encode_message<TestType>(ts, message, logtype, ir_buf));
-    reference_messages.push_back(message);
-    reference_timestamps.push_back(ts);
+    for (auto const& log_event : create_test_log_events()) {
+        // we ignore the utc offsets.
```

Comment:
If we update the deserialize_log_event methods as suggested, we could test the UTC offsets in this test case as well.